core/trino-main/src/main/java/io/trino/operator/scalar/ArrayDistinctFunction.java

```diff
@@ -31,16 +31,18 @@
 import it.unimi.dsi.fastutil.longs.LongSet;
 
 import static io.trino.operator.aggregation.TypedSet.createDistinctTypedSet;
+import static io.trino.operator.scalar.ArrayDistinctFunction.*;
 import static io.trino.spi.function.InvocationConvention.InvocationArgumentConvention.BLOCK_POSITION;
 import static io.trino.spi.function.InvocationConvention.InvocationReturnConvention.FAIL_ON_NULL;
 import static io.trino.spi.function.OperatorType.HASH_CODE;
 import static io.trino.spi.function.OperatorType.IS_DISTINCT_FROM;
 import static io.trino.spi.type.BigintType.BIGINT;
 
-@ScalarFunction("array_distinct")
+@ScalarFunction(NAME)
```

Comment:
revert this change. The `@ScalarFunction("<func-name>")` makes it easier to navigate the codebase IMO.

Reply:
Same in ArraySortFunction.